README.md

```diff
@@ -15,6 +15,8 @@ $ cpm [i|r|l|u|U|s|S|c|h] [pkg]...
 -> s|search  search for a package
 -> S|show    show information about a package
 -> I|info    same as show
+-> f|files   show file list of package
+-> f|from    show package which owns a file
```

Comment:
Split this up so that `f` is files and `F` is from.

Reply:
dont forget to add F to the list of options at line 8 in the readme, and update the script's help to reflect it

Reply:
i mean, its something i could see myself missing, just wanted to be a reminder

Reply:
The funny thing was how much you underestimated my stubbornness.

Reply:
#26